kahuna/public/js/components/gr-add-label/gr-add-label.html

```diff
@@ -1,5 +1,4 @@
-<button class="button-shy"
-        ng:class="{'small': ctrl.grSmall}"
+<button ng:class="{'small': ctrl.grSmall}"
```

Comment:
This [component](https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-add-label/gr-add-label.js#L47) does. Its possible misleading as small in this case means no button text, just the icon...